mysql-test/suite/sys_vars/t/show_slave_auth_info_grant.test

```diff
@@ -0,0 +1,9 @@
+--echo #
+--echo # MDEV-30189 Add remaining mariadbd replication command line options as system variables
+--echo #
+
+--let var = show_slave_auth_info
+--let grant = REPLICATION MASTER ADMIN
+--let value = TRUE
+
+--source suite/sys_vars/inc/sysvar_global_and_session_grant.inc
```

Comment:
I mention in another comment that we don't want to add any behavior to set this variable at run-time, only view it, so we don't need this test. Though I wanted to highlight it is a great find to use `sysvar_global_and_session_grant.inc`, so it would have been a great test otherwise :innocent: